Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM        prom/busybox:latest
-MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
+LABEL maintainer=The Prometheus Authors <prometheus-developers@googlegroups.com>
```

Comment:
Thanks for taking it! IIUC you need double quotes.

Reply:
Yes! I am also closing this PR. Will open up a new one